frontend/apps/filemanager/filemanagermenu.lua

```diff
@@ -175,6 +175,22 @@ function FileManagerMenu:setUpdateItemTable()
 
     -- info tab
     -- insert common info
+    table.insert(self.tab_item_table.info, {
+        text = _("Open last document"),
+        callback = function()
+            local last_file = G_reader_settings:readSetting("lastfile")
+            if last_file and lfs.attributes(last_file, "mode") ~= "file" or last_file == nil then
```

Comment:
This can be simplified to `if not last_file or lfs.attributes(last_file, "mode") ~= "file"`. What would be even better is to disable this menu item when `not last_file == true`.